sensor_msgs/Image picture
float32[] ranges
float32[] intensities
float32 angle_min
float32 angle_increment
geometry_msgs/Vector3 position
geometry_msgs/Quaternion rotation
